server.py

```diff
@@ -28,6 +28,20 @@ def get_latex():
             'error': str(e)
         }
 
+@app.route('/matrix-raw-echelon-form', methods=['POST'])
+def get_matrix_raw_echelon_form():
+    try:
+        latex(latex2sympy(request.json['data']).subs(variances).rref()[0])

```

Comment:
Why this line is duplicated here?

Reply:
Yes, this line here doesn't make sense. I removed it